doc/getting_started.mld

```diff
@@ -0,0 +1,78 @@
+
+{[
+dune build @fmt
+]}
+
+{2 Code style}
+
+There are a number of preset code style profiles, selected using the `--profile` option by passing `--profile=<name>` on the command line or adding `profile = <name>` to an .ocamlformat configuration file. Each profile is a collection of settings for all options, overriding lower priority configuration of individual options. So a profile can be selected and then individual options can be overridden if desired.
+
+The `conventional` or `default` profile aims to be as familiar and "conventional" appearing as the available options allow.
+
+The `ocamlformat` profile aims to take advantage of the strengths of a parsetree-based auto-formatter, and to limit the consequences of the weaknesses imposed by the current implementation. This is a style which optimizes for what the formatter can do best, rather than to match the style of any existing code. Instead of familiarity, the focus is on legibility, keeping the common cases reasonably compact while attempting to avoid confusing formatting in corner cases. General guidelines that have directed the design include:
+
+- Legibility, in the sense of making it as hard as possible for quick visual parsing to give the wrong interpretation, is of highest priority;
+- Whenever possible the high-level structure of the code should be obvious by looking only at the left margin, in particular, it should not be necessary to visually jump from left to right hunting for critical keywords, tokens, etc;
+- All else equal compact code is preferred as reading without scrolling is easier, so indentation or white space is avoided unless it helps legibility;
+- Attention has been given to making some syntactic gotchas visually obvious.
+
+If no profile is selected, the `default` one is used.
+
+{2 Options}
+
+The full options' documentation is available through `ocamlformat --help`.
+Options can be modified by the means of:
+- an .ocamlformat configuration file with an `option = VAL` line
+- using the `OCAMLFORMAT` environment variable: `OCAMLFORMAT=option=VAL,...,option=VAL`
+- an optional parameter on the command line
+- a global `[@@@ocamlformat "option=VAL"]` attribute in the processed file
+- an `[@@ocamlformat "option=VAL"]` attribute on an expression in the processed file
+
+`.ocamlformat` files in the containing and all ancestor directories for each input file are used, as well as the global .ocamlformat file defined in `$XDG_CONFIG_HOME/ocamlformat`. The global .ocamlformat file has the lowest priority, then the closer the directory is to the processed file, the higher the priority.
+
+When the option `--enable-outside-detected-project` is not set, .ocamlformat files outside of the project (including the one in `XDG_CONFIG_HOME`) are not read. The project root of an input file is taken to be the nearest ancestor directory that contains a .git or .hg or dune-project file. If no config file is found, formatting is disabled.
```

Comment:
Having this sentence in a negative form (when it's **not** set) makes it a bit confusing.

Reply:
I agree, done.